### ISSUE TITLE
Kobo n236 storage (32 gb sd card doesnt show all the storage available)

### ISSUE THREAD

**Issue by alisonkit** [Kobo-InkBox/inkbox#68]
So I flashed inkbox to this device and it runs fine, however I switched from a 4gb sd card to a new 32 gig for the install, and when I connect the device to my pc it can only see 500mb, does anyone know what's going on?

**Comment by Szybet** [comment]
Look up the wiki on the topic of expanding the onboard storage

**Comment by alisonkit** [comment]
I checked it out, followed the steps, and yet the onboard image doesn't seem to be expanding. After increasing the partition (to 29gb), and expanding the size of the onboard file, when I look at it, it does show up as the file size I intend it to be (26gb), but once I plug in the device, it still only registers as having 511mb available storage. I used the method where you tell qemu-img to resize it.
